### ISSUE TITLE
Mapstore support in Making Map View Page

### ISSUE THREAD

**Issue by msilikale05** [resilienceacademy/resilience-academy-geonode#17]
This was suggested during the training in January in Finland as the way to complete the advancement of the map view page (this was also listed in the issues **"[Map View page #16](https://github.com/resilienceacademy/resilience-academy-geonode/issues/16)"**). Therefore, it was suggested that Geosolutions to support the RA Team to accomplish this with Mapstore by providing either an online training (Webinar) or provide with the necessary documentation that will help the RA Team to accomplish the map view feature in the Climate Risk Database.

**Comment by simboss** [comment]
@msilikale05 as per our discussion, I will send next week the agenda of the training we have in mind together with some dates for it.

It is partially based on a similar training we are doing next week, so the preparation time should be be small on our side. We are going to propose 3 to 4 webinar sessions of 3.5 hours.

In terms of the timezone I don't think that matters, since both Tanzania and Finland are in the same timezone right?

**Comment by msilikale05** [comment]
Yes, at the moment Tanzania and Finland have the same time. So great.
